cypress/automated-tests/youth-pass/file-type-upload.spec.js

```diff
@@ -0,0 +1,112 @@
+        cy.get('#element103').type(applicantCity).blur();
+        cy.get('#element154').type(applicantZipCode).blur();
+        cy.get('#element122_Option_1').click().blur();
+        cy.get('#form-section-6 > .form-section-buttons > .form-section-next').click();
+    });
+    
+    it('uploads word, gif, html, and jpeg file types to proof of age', () => {
+        cy.get('#element114_Option_1').click().blur();
+        cy.get('#element133').attachFile('youth-pass-test-file.docx')
+        cy.get('.k-text-error').should('exist');
+        cy.get('#form-element-wrapper_133').within(() => {
+            cy.get('.k-upload-status > .k-button').click();
+        })
+        cy.get('.k-text-error').should('not.exist');
+
+        cy.get('#element133').attachFile('youth-pass-test-image.gif')
+        cy.get('.k-text-success').should('exist');
+        cy.get('#form-element-wrapper_133').within(() => {
+            cy.get('.k-upload-status > .k-button').click();
+            cy.get('.k-text-success').should('not.exist');
+        })
+        
+        cy.get('#element133').attachFile('youth-pass-test-file.html')
+        cy.get('.k-text-error').should('exist');
+        cy.get('#form-element-wrapper_133').within(() => {
+            cy.get('.k-upload-status > .k-button').click();
+            cy.get('.k-text-error').should('not.exist');
+        })
+
+        cy.get('#element133').attachFile('youth-pass-test-image.jpeg')
+        cy.get('.k-text-success').should('exist');
+        cy.get('#form-section-7 > .form-section-buttons > .form-section-next').click();
+    });
+      
+    it('uploads js, jpg, json, and pdf file types to proof of address', () => {
+        cy.get('#element135_Option_1').click().blur();
+        cy.get('#element39').attachFile('youth-pass-test-file.js')
+        cy.get('.k-text-error').should('exist');
+        cy.get('#form-element-wrapper_39').within(() => {
+            cy.get('.k-upload-status > .k-button').click();
+            cy.get('.k-text-error').should('not.exist');
+        })
+
+        cy.get('#element39').attachFile('youth-pass-test-image.jpg')
+        cy.get('.k-text-success').should('exist');
+        cy.get('#form-element-wrapper_39').within(() => {
+            cy.get('.k-upload-status > .k-button').click();
+            cy.get('.k-text-success').should('not.exist');
+        })
+
+        cy.get('#element39').attachFile('youth-pass-test-file.json')
+        cy.get('.k-text-error').should('exist');
+        cy.get('#form-element-wrapper_39').within(() => {
+            cy.get('.k-upload-status > .k-button').click();
+            cy.get('.k-text-error').should('not.exist');
+        })
+
+        cy.get('#element39').attachFile('youth-pass-test-image.pdf')
+        cy.get('.k-text-success').should('exist');
+        cy.get('#form-section-8 > .form-section-buttons > .form-section-next').click();
+    });
+
+    it('uploads rtf and xlsx file types to proof of program enrollment', () => {
```

Comment:
If I'm reading right, I think this test description should probably say something about not accepting these types of uploads, right?

Reply:
Good call. I'm not quite sure how the `.DS_Store` snuck in there, I'll remove it and update the `it` blocks to more accurately describe which file types should and should not upload successfully